tests/acceptance/project-test.js

```diff
@@ -6,6 +6,16 @@ import { setupApplicationTest } from "ember-qunit";
 import { authenticateSession } from "ember-simple-auth/test-support";
 import { module, test } from "qunit";
 
+let _local_assert = { step: () => {} };
+
+class NotifyServiceStub extends Service {
+  error() {
+    _local_assert.step("error");
+  }
+  // Needed for mocking, throws error otherwise
+  setTarget() {}
+}
```

Comment:
Could your stubs not be replaced by sinon?

Reply:
Which package? They all seem a little abandoned. :disappointed: 

I like the manual stubbing, since it doesn't add any other dependency / complexity. It's all pure ember.

Reply:
In a lot of our projects `ember-sinon-qunit` is installed. They also have useful call assertions and stuff. What im not a huge fan of is the `_local_assert.step` . It seems very hacky. With a sinon you could just assert against the stub

Reply:
You got me convinced by pointing to `_local_assert.step`. This is a marker of bad testing setup. ;-) 

I setup sinon and spyed on the `this.notify.error` call. Way cleaner! Thanks!